src/stories/0-Welcome.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+// import { linkTo } from '@storybook/addon-links';
+// import { Welcome } from '@storybook/react/demo';
+import PageTitle from '../components/PageTitle';
+
+export default {
+  title: 'Welcome',
```

Comment:
Can we change this name and the file name away from the example and reflect the Story here?

Reply:
Need to change "title: 'Welcome'" as well?

Reply:
It should have the component name so that its easy to navigate as we add more stories.

Reply:
ok thanks

Reply:
change the title with component name